src/parser/type_checking.c

```diff
@@ -24378,7 +24378,7 @@ static PT_TYPE_ENUM
 pt_wrap_type_for_collation (const PT_NODE * arg1, const PT_NODE * arg2, const PT_NODE * arg3,
 			    PT_TYPE_ENUM * wrap_type_collection)
 {
-  PT_TYPE_ENUM common_type = PT_TYPE_CHAR;
+  PT_TYPE_ENUM common_type = (PT_IS_COLLECTION_TYPE (arg1->type_enum))?PT_TYPE_CHAR:PT_TYPE_VARCHAR;
```

Comment:
ok, I got it.
please fix indent before merging.
```suggestion
  PT_TYPE_ENUM common_type = (PT_IS_COLLECTION_TYPE (arg1->type_enum)) ? PT_TYPE_CHAR : PT_TYPE_VARCHAR;
```

Reply:
Ok,